llama_index/evaluation/pairwise.py

```diff
@@ -13,36 +14,38 @@
 )
 from llama_index.prompts.mixin import PromptDictType, PromptMixin, PromptMixinType
 
-DEFAULT_SYSTEM_TEMPLATE = """
-You are an expert evaluation system for a question answering chatbot.
-
-You are given the following information:
-- a user query,
-- Answer 1
-- Answer 2
-
-
-Your job is to output whether Answer 1 is better, or Answer 2 is better, or
-they are equally good at answering the user query.
-
-Output "1" if Answer 1 is better, "2" if Answer 2 is better, and \
-    "TIE" if they are equally good.
-
-Please output two lines:
-- first line: "1", "2", or "TIE"
-- second line: a short explanation for your decision.
-"""
-
-DEFAULT_USER_TEMPLATE = """
-## User Query
-{query}
-
-## Answer 1
-{reference_answer}
+DEFAULT_SYSTEM_TEMPLATE = (
```

Comment:
curious: is the multi-line string concat preferred over the triple quote?

Reply:
I prefer triple quote as it's easier to write, but I don't believe black formatter picks these up. It can only pick up the single quote or these implicit multi-line ones. I believe black formats really long single quote strings using this implicit style as well.

Reply:
I can change to """ version if that is preferred 😎

Reply:
no worries, I don't want to be pedantic either way. was just curious why one vs. the other.